extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.css

```diff
@@ -115,3 +115,30 @@ amp-story-dev-tools-tab-logs {
l-opacity=".1"/><rect x="187" y="158.1" width="10.4" height="10.4" rx="5.2" fill="#79B3FF"/><path d="M260 254c0-4-2-8-5-8l-11 4 1 17 11-5c2 0 4-4 4-8z" fill="#FCF75A"/><path d="M227 247c2-2 3-3 5-3l-31-22c-2-1-4-1-5 1-2 2-1 4 0 5l30 25 1-6z" fill="#4E5EAF"/><path d="M242 258c2-2 4-5 1-6l-11-8c-2 0-3 1-5 3l-1 6 16 13c3 2-1-6 0-8z" fill="#FCF75A"/><path d="M244 267c3 0 5-4 5-9 0-4-2-8-5-8s-5 4-5 8c0 5 2 9 5 9z" fill="#AE9D00"/><path d="M247 258c0-3-1-5-3-5-1 0-3 2-3 5l2 4 1-1-1-3v-3l1-1 1 1 1 3c0 3-2 7-4 8l1 1c2-2 4-5 4-9z" fill="#FCF75A"/><g filter="url(#filter3_d)"><path d="M123 66c2 4-6 12-18 17s-23 6-25 1c-2-4-7-76-7-76s48 53 50 58z" fill="#4E5EAF"/><path d="M104 52c-9 4-19 5-27 3l1 13a46 46 0 0042-7l-11-12-5 3zM88 25c-4 1-9 2-14 1l1 15c8 1 17 0 25-3L88 25z" fill="#9F9AEE"/></g><defs><filter id="filter0_d" x="108.3" y="211.6" width="48.9" height="50.7" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="6"/><feGaussianBlur stdDeviation="8"/><feColorMatrix values="0 0 0 0 0.0375 0 0 0 0 0.045 0 0 0 0 0.1125 0 0 0 0.56 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter1_d" x="133.7" y="52.6" width="52.1" height="54.7" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="6"/><feGaussianBlur stdDeviation="8"/><feColorMatrix values="0 0 0 0 0.0375 0 0 0 0 0.045 0 0 0 0 0.1125 0 0 0 0.56 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter2_d" x="1.3" y="110.3" width="56.7" height="58.3" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="4"/><feGaussianBlur stdDeviation="8"/><feColorMatrix values="0 0 0 0 0.0375 0 0 0 0 0.045 0 0 0 0 0.1125 0 0 0 1 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter><filter id="filter3_d" x="57" y="0" width="74" height="103.1" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dx="-4" dy="4"/><feGaussianBlur stdDeviation="6"/><feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter></defs></svg>') !important;
```

Comment:
There is a confetti looking artifact that differs from the mocks.
screen graphic and font weight differs from mock as well. 

<img width="732" alt="Screen Shot 2021-01-14 at 4 08 26 PM" src="https://user-images.githubusercontent.com/3860311/104649474-cb364200-5682-11eb-8e14-680425fc2f23.png">

Reply:
Font weight is a tricky one, we don't import the semibold weight into the document, but maybe it makes sense to change the import from bold to semibold since bold is not used much